components/ws-daemon/nsinsider/pkg/nsenter/nsexec.c

```diff
@@ -70,33 +72,41 @@ int setns(int fd, int nstype)
 }
 #endif
 
-static void write_log_with_info(const char *level, const char *function, int line, const char *format, ...)
+static void write_log(const char *level, const char *format, ...)
```

Comment:
So far the `nsexec.c` was unmodified which means updating it to newer versions from runc is easy(ier). Are the modifications worth giving up that benefit, or can we get by with less/no modifications?

Reply:
The upstream version is quite [different now](https://github.com/opencontainers/runc/blob/master/libcontainer/nsenter/nsexec.c) and the change is from the there (mostly)

Reply:
excellent point